src/components/__tests__/constants.test.ts

```diff
@@ -1,4 +1,4 @@
-import { KEYS } from '../../components/constants'
+import { KEYS } from '../constants'
 
 console.log(KEYS)
```

Comment:
This test file isn't super valuable, might as well ditch it (even as a snapshot test, it would just assert that a constant is as defined in the constants file, which is asserting that the language works as expected).

Reply:
@mcataford 👌 Right-o. Dropped in [`38f069b`](https://github.com/patw0929/react-intl-tel-input/pull/392/commits/38f069b5be85ed7a4fa386292a90bdd225fe0f66).